src/index.html

```diff
@@ -17,16 +17,30 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search-form"
+      action="#"
+      data-qa="big"
+    >
```

Comment:
Remember to add empty lines between multiline sibling blocks of HTML. In this case, there should be an empty line between the closing </form> tag on line 31 and the opening <form> tag on line 33.